core/domain/user_services.py

```diff
@@ -668,7 +668,8 @@ def _update_user_contribution_rights(
         user_contribution_rights: UserContributionRights. The updated
             UserContributionRights object of the user.
     """
-    if user_contribution_rights.can_review_at_least_one_item():
+    if user_contribution_rights.can_review_at_least_one_item() or (
+       user_contribution_rights.can_submit_at_least_one_item()):
```

Comment:
I think the docstring needs to be updated?

Reply:
Added